plugin/pkg/scheduler/algorithm/types.go

```diff
@@ -117,6 +117,12 @@ type ReplicaSetLister interface {
 	GetPodReplicaSets(*v1.Pod) ([]*extensions.ReplicaSet, error)
 }
 
+// PersistentVolumeClaimLister interface represents anything that can list PVCs for a scheduler.
+type PersistentVolumeClaimLister interface {
```

Comment:
Why do we need this interface? Why can't we pass a PVC Lister to scheduler?

Reply:
We don't need this interface, but looking around scheduler code, it seems to be a generic pattern to wrap everything in standalone interface instead of using listeners. I am just following the suit.

I'm removing the interface in 2nd commit so we can either squash it or delete it.